openshift/resolve-yamls.sh

```diff
@@ -53,3 +53,16 @@ function resolve_resources() {
     echo >>$resolved_file_name
   done
 }
+
+function generate_pipeline_resources() {
+    local output_file=$1
+    local image_prefix=$2
+    local image_tag=$3
+
+    resolve_resources config/ $output_file noignore $image_prefix $image_tag
+
+    # Appends addon configs such as prometheus monitoring config
+    for yaml in $(find openshift/addon-config/ -name "*yaml"); do
+      cat ${yaml} >> $output_file
```

Comment:
shouldn't we insert `---` after each of these yaml files?

Reply:
I think those is for a rewrite of resolve-yamls which could be addressed in a different review,  here @hrishin is only add a feature to an exiting script, let's introduce feature/improvement/bugs in different prs, it would make it easier to pinpoint,

Reply:
actually nevermind my comment, this needs an improvement of the full function, 

@hrishin you are wrapping `resolve_resources` around `generate_pipelines` why don't you just improve `resolve_resources`  and use directly ? I thin that's what you did in a previous PR ?

(and please rebase/amend your commit)

Reply:
@chmouel yes initially we tried to improve  `resolve_resources` but it's being used for generally at other places e.g. https://github.com/openshift/tektoncd-pipeline/blob/master/openshift/e2e-tests-openshift.sh#L56-L59

hence wrapping it

Reply:
@chmouel marking it as resolved, feel free to open it again.